include/tss2/tss2_policy.h

```diff
@@ -110,6 +110,7 @@ typedef TSS2_RC (*TSS2_POLICY_CB_EXEC_AUTH) (
     TPM2B_NAME *name,
     ESYS_TR *object_handle,
     ESYS_TR *auth_handle,
+    bool *flush_handle,
```

Comment:
We add this here but then don't actually make use of this in the generic policy callbacks and only in the IFAPI portions of the code. Shouldn't this be in either the userdata pointer within the ifapi_policyeval_cbauth (which is a FAPI_CONTEXT) or within one of the other context structures:
```C
    IFAPI_POLICY_EXEC_CTX *current_policy;
    IFAPI_POLICY_EXEC_CB_CTX *cb_ctx;
```

Reply:
Yes that's true. I will move `flush_handle` to the context. I wanted to add another parameter  `auth_object` to the callback to enable the usage of the callback for  PolicySigned with symmetric keys. Here only the key has to be loaded and the authorization is only needed for parent objects. If TSS2_POLICY_CB_EXEC_AUTH should not be changed I will add a new callback TSS2_POLICY_CB_EXEC_LOAD. Would this be OK or do you see any problems?

Reply:
> Yes that's true. I will move `flush_handle` to the context. I wanted to add another parameter `auth_object` to the callback to enable the usage of the callback for PolicySigned with symmetric keys. Here only the key has to be loaded and the authorization is only needed for parent objects. If TSS2_POLICY_CB_EXEC_AUTH should not be changed I will add a new callback TSS2_POLICY_CB_EXEC_LOAD. Would this be OK or do you see any problems?

That seems fine

Reply:
> > Yes that's true. I will move `flush_handle` to the context. I wanted to add another parameter `auth_object` to the callback to enable the usage of the callback for PolicySigned with symmetric keys. Here only the key has to be loaded and the authorization is only needed for parent objects. If TSS2_POLICY_CB_EXEC_AUTH should not be changed I will add a new callback TSS2_POLICY_CB_EXEC_LOAD. Would this be OK or do you see any problems?
> 
> That seems fine

[This](https://github.com/tpm2-software/tpm2-tss/commit/067dc2a028e131f11f40e37d072c4219e840a5ac) patch seems simpler than whats described and it fixes the problem? Do you have a test?

Reply:
> [067dc2a](https://github.com/tpm2-software/tpm2-tss/commit/067dc2a028e131f11f40e37d072c4219e840a5ac) patch seems simpler than whats described and it fixes the problem? Do you have a test?

I have tested PolicySecret with a key by using the command line tools. I will backport the PR and can create an integration test afterwards.